packages/fx-core/.eslintignore

```diff
@@ -1,2 +1,4 @@
 templates/plugins/resource/spfx/*
-src/plugins/resource/spfx/authCode.ts
\ No newline at end of file
+src/plugins/resource/spfx/authCode.ts
+src/plugins/solution/fx-solution/arm.ts
```

Comment:
temporary ignore the 2 files to avoid eslint error, which caused by required API not in TeamsFx-API plugin interface. In the future the API will be added to plugin interface and this change can be reverted.

Reply:
@LongOddCode just help found a better way for my situation, I will try his approach and update this PR later.